sagemaker-training-compiler/tensorflow/single_gpu_single_node/vision-transformer.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
<p>Related question,</p><p> </p><p>If you'll fix the TensorFlow estimator to use the <code>framework_version</code> and <code>py_version</code> instead of using the URI directly, will you remove <code>TRCOMP_IMAGE_URI</code>? </p><p> </p><p>And this subsection is only about setting up that couple of variables, and not a complete section for Training Setup. EPOCHS can be simply defined in the next code block along with other variables where you have the following:</p><pre class="ql-syntax" spellcheck="false">BATCH_SIZE = 64
LEARNING_RATE = 1e-3
WEIGHT_DECAY = 1e-4
</pre>

Reply:
Yes I will remove `TRCOMP_IMAGE_URI` once the PySDK is released.

Reply:
@mchoi8739  I would like to maintain the 'Training Setup' section for the sake for uniformity. Our upcoming multi node notebook will have other cluster setup information like NUM_INSTANCES. Do you see an issue with this ?

Reply:
I think all of the variables can be in the same code sample. I don't know how the next notebook would look like, so probably I'll wait for your next notebook. But to me still I think there's no need to make extra section that can be looking like an extra step. 

How about changing the title to "Set Up Variables for SageMaker Training"? "Training Setup" sounds like a lot greater topic than specifying some variables.

Reply:
Fixed.